be/src/exprs/aggregate_functions.cpp

```diff
@@ -341,6 +339,15 @@ void AggregateFunctions::decimalv2_avg_update(FunctionContext* ctx,
     ++avg->count;
 }
 
+StringVal AggregateFunctions::decimalv2_avg_serialize(
+        FunctionContext* ctx, const StringVal& src) {
+    StringVal result = string_val_get_value(ctx, src);
+    if (!src.is_null) {
```

Comment:
Is this check needed? is_null has been initialized false, I think there is no need to check it. So does the finalize function. You can generate StringVal instead of call string_val_get_value to avoid a if branch.